fast_forward/bonded_functions.py

```diff
@@ -51,12 +51,11 @@ def _vs3fd_func(pars, x, positions):
     return x - r_predicted_vs
 
 def _vs3fd(arr1, arr2, arr3, arr4):
-    frames = arr1.shape[0]
     fit_params = create_params(a=1, b=1)
     pos_list = [arr2 / 10, arr3 / 10, arr4 / 10] # convert to nm
     fit = minimize(_vs3fd_func, fit_params, args=(arr1/10, pos_list,))
     ab = np.array([fit.params["a"].value, fit.params["b"].value])
-    return np.tile(ab, frames).reshape((frames, 2))
+    return ab
```

Comment:
@csbrasnett sorry you missunderstood: We need the `np.tile(ab, frames)` just not the reshape

Reply:
we need it because the time-series needs to have the shape (n-frames, values). Since we use the same workflow for VS and the other interactions. The proper way would be to clean this up and make a separate workflow for the VS. So add a method to the interactions fitter that deals with this completely aside from the rest. 

If I find time I'll do it in a separate PR. For now let's keep the tile as it is working